Cargo.toml

```diff
@@ -21,6 +21,20 @@ resolver = "2"
 
 # `cargo release` settings
 
+[workspace.dependencies]
+incrementalmerkletree = "0.6.0"
+orchard = "0.9.0"
+sapling-crypto = "0.2.0"
```

Comment:
These dependency versions have been bumped, which seems fine if the tests pass, we trust the ECC dependencies.

Reply:
Ah, yes. Should've mentioned that in the PR description.